<launch>
  <node pkg="ardrone_pkg" type="ardrone_action_client.py" name="ardrone_action_client"  output="screen">
  </node>
</launch>